rust/crd/src/lib.rs

```diff
@@ -29,6 +29,7 @@ use std::collections::BTreeMap;
 use strum::{Display, EnumIter, EnumString};
 
 pub const APP_NAME: &str = "kafka";
+pub const OPERATOR_NAME: &str = "kafka.stackable.tech";
```

Comment:
Isn't the operator name `kafka-operator` ?

Reply:
What would you suggest? This is consistent for all the bumps and labels in other operators. Not to kin to change this everywhere?